Makefile

```diff
@@ -18,6 +23,14 @@ PACKAGE_VENDOR := Brave New Software Project, Inc
 PACKAGE_MAINTAINER := Lantern Team <team@getlantern.org>
 PACKAGE_URL := https://www.getlantern.org
 
+GH_USER := getlantern
+#GH_USER := xiam
```

Comment:
I still need this here for testing on another namespace without worrying of messing our official repo.